libs/ui/src/tokens/components/atoms/_textarea.css

```
@@ -1,91 +1,131 @@
+    oklch(0.681 0.142 75.834),
+    oklch(0.795 0.162 86.047)
+  );
+  --color-textarea-border-warning-focus: var(
+    --color-textarea-border-warning-base
   );
-  --color-textarea-border-danger-focus: var(--color-textarea-danger);
+
+  /* === Placeholder-on-danger (not in Figma) === */
   --color-textarea-placeholder-danger: --alpha(
-    var(--color-textarea-danger) /
+    var(--color-textarea-border-danger-base) /
```

Stylelint (17.12.0)

[error] Unexpected newline after "/" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)